code/datums/components/uplink.dm

```diff
@@ -168,19 +168,28 @@
 	if(uplink_handler.has_objectives)
 		var/list/primary_objectives = list()
 		for(var/datum/objective/task as anything in uplink_handler.traitor_datum?.objectives)
-			primary_objectives += task.explanation_text
+			var/list/task_data = list()
+			if(length(primary_objectives) > length(GLOB.phonetic_alphabet))
+				task_data["task_name"] = "DIRECTIVE [primary_objectives.len]" //The english alphabet is WEAK
```

Comment:
why not use len + 1 if you were using it in the other one. also we prefer length(list) to list.len

Reply:
Oversight, thanks. Has been fixed.